KNNCodecTestUtil broken due to fieldInfo constructor

https://github.com/opensearch-project/k-NN/actions/runs/3192735420/jobs/5210547276

```
> Task :compileTestJava
/home/runner/work/k-NN/k-NN/src/test/java/org/opensearch/knn/index/codec/KNNCodecTestUtil.java:167: error: constructor FieldInfo in class FieldInfo cannot be applied to given types;
            return new FieldInfo(
                   ^
  required: String,int,boolean,boolean,boolean,IndexOptions,DocValuesType,long,Map<String,String>,int,int,int,int,VectorEncoding,VectorSimilarityFunction,boolean
  found: String,int,boolean,boolean,boolean,IndexOptions,DocValuesType,long,Map<String,String>,int,int,int,int,VectorSimilarityFunction,boolean
  reason: actual and formal argument lists differ in length
```

--- a/src/test/java/org/opensearch/knn/index/codec/KNNCodecTestUtil.java
+++ b/src/test/java/org/opensearch/knn/index/codec/KNNCodecTestUtil.java
@@ -21,6 +21,7 @@ import org.apache.lucene.index.SegmentWriteState;
 import org.apache.lucene.index.SortedDocValues;
 import org.apache.lucene.index.SortedNumericDocValues;
 import org.apache.lucene.index.SortedSetDocValues;
+import org.apache.lucene.index.VectorEncoding;
 import org.apache.lucene.index.VectorSimilarityFunction;
 import org.apache.lucene.search.Sort;
 import org.apache.lucene.store.ChecksumIndexInput;
@@ -178,6 +179,7 @@ public class KNNCodecTestUtil {
                 pointIndexDimensionCount,
                 pointNumBytes,
                 vectorDimension,
+                VectorEncoding.FLOAT32,
                 vectorSimilarityFunction,
                 softDeletes
             );

